can_tools/scrapers/__init__.py

```diff
@@ -1,4 +1,10 @@
 from can_tools.scrapers.base import CMU, DatasetBase
+
+# Is there any reason to pipe the imports through all the different layers? seems
+# like it's just as easy to add the one import directly to the scraper here rather than
+# having to include in a couple different __init__.py's.
+from can_tools.scrapers.official.MS.ms_vaccine import MSCountyVaccine
```

Comment:
Good question -- it is kinda annoying. I think we could probably just include it here as you outlined